modules/uhttp/http.go

```diff
@@ -61,6 +61,7 @@ const (
 )
 
 var _ service.Module = &Module{}
+var tracingKey = struct{}{}
```

Comment:
nit: group these `var`s:

```
var (
   _ service.Module = &Module{}

   tracingKey = struct{}{}
}
```

also, why is tracingKey the empty struct? shouldn't it have some value?